plugins/modules/zos_copy.py

```diff
@@ -633,12 +734,17 @@ def copy_to_seq(
             rc = datasets.copy(new_src, dest)
             # *****************************************************************
             # When Copying a PDSE member to a non-existent sequential data set
-            # using: cp "//'SOME.PDSE.DATA.SET(MEMBER)'" "//'SOME.DEST.SEQ'"
+            # using cp "//'SOME.PDSE.DATA.SET(MEMBER)'" "//'SOME.DEST.SEQ'",
             # An I/O abend could be trapped and can be resolved by allocating
             # the destination data set before copying.
             # *****************************************************************
             if rc != 0:
-                self._allocate_ps(dest, alloc_vol=alloc_vol)
+                sz = None
+                if space_primary is not None:
```

Comment:
In this case its data set copied to data set, you have the data set src (and its size), so why not create the dest using the src as input to model_ds code and only fall back to creating a new data set when the user specifies the options.

Reply:
I'm not sure how to get the size of a PDSE member, and whether we need to add to that value for a PDS/sequential allocation.  Who could I use as a resource for this?

Reply:
Good question, after we talked I started to question how i never see a size for a member, probably I/you can ask Mike Fulton, likely at the C level we can calculate this but see it as a low priority now and more a nice to have. If their is an easy way to do it, lets get that else we can Backlog this wish list item.

Reply:
I recall we decided to go with the default for now of 5mb which is also the same as our ZOAU dependency. There is enough information here and actionable responses if the space is not enough to not warrant any more analysis to obtain the pre existing member size (for now)